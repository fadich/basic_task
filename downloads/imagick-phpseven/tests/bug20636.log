
---- EXPECTED OUTPUT
success
---- ACTUAL OUTPUT
php7.0: unable to acquire cache view `No such file or directory' @ fatal/cache-view.c/AcquireAuthenticCacheView/121.
---- FAILED
