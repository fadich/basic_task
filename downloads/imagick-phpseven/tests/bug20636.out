php7.0: unable to acquire cache view `No such file or directory' @ fatal/cache-view.c/AcquireAuthenticCacheView/121.